src/anubhav/NestedSwitchExamples/BankOperationsNestedSwitch.java

```diff
@@ -0,0 +1,158 @@
+package anubhav.NestedSwitchExamples;
+
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                          Total debit operation - 5 times
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2 */
+
+public class BankOperationsNestedSwitch {
+	
+	int iciciDebitCount, iciciCreditCount;  
+	int hdfcDebitCount,  hdfcCreditCount;   
+	int bobDebitCount,   bobCreditCount;    
+	int sbiDebitCount,   sbiCreditCount; 	 
+	double hdfcBal=100000; 
+	double iciciBal=100000; 
+	double bobBal=100000; 
+	double sbiBal=100000;
+	double totalAmount;
+	//double iciciLBalance;
+	//double hdfcLBalance;
+	//double bobLBalance;
+	//double sbiLBalance;	
+	
+	void operationDebitCredit(String operation, String bankName, double amount) {
+		operation= operation.toUpperCase();
+		bankName=bankName.toUpperCase();
+		switch (operation) {
+			case "DEBIT":
+				switch (bankName) {
+					case "ICICI":
+						if (amount>iciciBal) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method